ffv1.md

```diff
@@ -214,6 +227,27 @@ $$b=Cb+g$$
 
 [@!ISO.15444-1.2016]
 
+An FFV1 frame using JPEG2000-RCT MUST use one of the following arrangements:
+
+- Y, Cb, Cr
+- Y, Cb, Cr, Alpha
+
+When FFV1 uses the JPEG2000-RCT colorspace, the horizontal lines are interleaved to improve caching efficiency since it is most likely that the RCT will immediately be converted to RGB during decoding. The interleaved coding order is also Y, then Cb, then Cr, and then if used Alpha.
+
+As an example, a frame that is two pixels wide and two pixels high, could be comprised of the following structure:
+
++------------------------+------------------------+
+| Pixel[1,1]             | Pixel[2,1]             |
+| Y[1,1] Cb[1,1] Cr[1,1] | Y[2,1] Cb[2,1] Cr[2,1] |
++------------------------+------------------------+
+| Pixel[1,2]             | Pixel[2,2]             |
+| Y[1,2] Cb[1,2] Cr[1,2] | Y[2,2] Cb[2,2] Cr[2,2] |
++------------------------+------------------------+
+
+In JPEG2000-RCT colorspace, the coding order would be left to right and then top to bottom, with values interleaved by lines and stored in this order:
+
+Y[1,1] Y[2,1] Cb[1,1] Cb[2,1] Cr[1,1] Cr[2,1] Y[1,2] Y[2,2] Cb[1,2] Cb[2,2] Cr[1,2] Cr[2,2]
+
 ## Coding of the sample difference
 
 Instead of coding the n+1 bits of the sample difference with Huffman-, or Range coding (or n+2 bits, in the case of RCT), only the n (or n+1) least significant bits are used, since this is sufficient to recover the original sample. In the equation below, the term "bits" represents bits_per_raw_sample+1 for RCT or bits_per_raw_sample otherwise:
```

Comment:
Is the dash after `Huffman-` needed in English? `Huffman or Range coding` looks better to me.